plans/main.fmf

```diff
@@ -0,0 +1,59 @@
+summary: Run Python Podman Tests
+
+discover:
+    how: fmf
+execute:
+    how: tmt
+prepare:
+  - name: pkg dependencies
+    how: install
+    package:
+      - make
+      - podman
+      - python3-pip
+
+  - name: pip dependencies
+    how: shell
+    script:
+      - pip3 install -r test-requirements.txt
+
+  - name: ssh configuration
+    how: shell
+    script:
+        - ssh-keygen -t ecdsa -b 521 -f /root/.ssh/id_ecdsa -P ""
+        - cp /root/.ssh/authorized_keys /root/.ssh/authorized_keys%
+        - cat /root/.ssh/id_ecdsa.pub >>/root/.ssh/authorized_keys
+
+/upstream:
+    /lint:
+        summary: Run Python Podman Tests on upstream PRs
+        discover+:
+            test: /tests/lint
+
+    /integration_tests:
+        summary: Run Python Podman Tests on upstream PRs
+        discover+:
+            test: /tests/all_integration
+
+    /unittest:
+        summary: Run Python Podman Unit Tests on upstream PRs
+        discover+:
+            test: /tests/unittest
+
+    /integration:
```

Comment:
just confirming that `/tests/integration` and `/tests/all_integration` are different.

Reply:
yeah, bad naming maybe. /integration runs coverage for integration tests on a single python version while /all_integrations runs the actual tests on all python versions